docs/spec/overview.md

```diff
@@ -9,9 +9,7 @@ backing implementations (much like the Kubernetes Ingress resource).
 - A **Subscription** describes the transformation of an event and optional
   forwarding of a returned event.
 
-- A **Source** allows an incoming events from an external system to be
-  _Subscribable_. A _Subscription_ is used to connect these events to
-  subsequent processing steps.
+- A **Source** emits incoming events to a _Channel_.
 
 - A **Channel** provides event persistance and fanout of events from a
   well-known input address to multiple outputs described by _Subscriptions_.
```

Comment:
I think with this change, the drawing in https://docs.google.com/drawings/d/10mmXzDb8S_4_ZG_hcBr7s4HPISyBqcqeJLTXLwkilRc/edit    needs update too